extensions-core/azure-extensions/src/test/java/org/apache/druid/data/input/azure/AzureStorageAccountInputSourceTest.java

```diff
@@ -121,33 +127,42 @@ public void setup()
     EasyMock.expect(azureAccountConfig.getAccount()).andReturn(DEFAULT_STORAGE_ACCOUNT).anyTimes();
   }
 
-  @Test(expected = IllegalArgumentException.class)
+  @Test
   public void test_constructor_emptyUrisEmptyPrefixesEmptyObjects_throwsIllegalArgumentException()
   {
     replayAll();
-    azureInputSource = new AzureStorageAccountInputSource(
-        entityFactory,
-        azureCloudBlobIterableFactory,
-        inputDataConfig,
-        azureAccountConfig,
-        EMPTY_URIS,
-        EMPTY_PREFIXES,
-        EMPTY_OBJECTS,
-        null,
-        azureStorageAccountInputSourceConfig,
-        null
+
+    //noinspection ResultOfObjectAllocationIgnored
+    assertThrows(
+        IllegalArgumentException.class,
+        () -> new AzureStorageAccountInputSource(
+            entityFactory,
+            azureCloudBlobIterableFactory,
+            inputDataConfig,
+            azureAccountConfig,
+            EMPTY_URIS,
+            EMPTY_PREFIXES,
+            EMPTY_OBJECTS,
+            null,
+            azureStorageAccountInputSourceConfig,
+            null
+        )
     );
   }
 
   @Test
   public void test_createEntity_returnsExpectedEntity()
   {
-    EasyMock.expect(entityFactory.create(EasyMock.eq(CLOUD_OBJECT_LOCATION_1), EasyMock.anyObject(AzureStorage.class), EasyMock.eq(AzureStorageAccountInputSource.SCHEME))).andReturn(azureEntity1);
+    EasyMock.expect(entityFactory.create(
+        EasyMock.eq(CLOUD_OBJECT_LOCATION_1),
+        EasyMock.anyObject(AzureStorage.class),
+        EasyMock.eq(AzureStorageAccountInputSource.SCHEME)
+    )).andReturn(azureEntity1);
```

Comment:
Nit: more readable this way.
```suggestion
    EasyMock.expect(
        entityFactory.create(
            EasyMock.eq(CLOUD_OBJECT_LOCATION_1),
            EasyMock.anyObject(AzureStorage.class),
            EasyMock.eq(AzureStorageAccountInputSource.SCHEME)
        )
    ).andReturn(azureEntity1);
```